### ISSUE TITLE
GitHub Action for Lighthouse

### ISSUE THREAD

**Issue by ann-kilzer** [WomenInSoftwareEngineeringJP/home#5]
Set up Google Lighthouse as a GH Action so we can monitor a11y and SEO

https://developer.chrome.com/docs/lighthouse/overview/

**Comment by RossellaFer** [comment]
The [Github action](https://github.com/GoogleChrome/lighthouse-ci/tree/main) works but 15 tests are failing, so I am not creating a PR yet because the build will always fail. We could gradually implement checks using the assert configuration ([documentation](https://github.com/GoogleChrome/lighthouse-ci/tree/main)) to exclude the failed checks that we cannot address yet. 

### How to test locally

To test in local environment, run the following commands:
```
npm install && npm install -g @lhci/cli@0.13.x
npm run build
lhci autorun
```

### Notes on the files created 

The `lhci autorun` command is the one that actually runs the Lighthouse checks, following the configuration in `lighthouserc.json`. The config file was necessary since by default the checks run against an `localhost:automaticallygeneratedport/index.html` path.
